tests/main.cpp

```diff
@@ -97,3 +97,21 @@ TEST_CASE("Test getLatestBlock") {
   CHECK(!blockHash.publicKey.toBase58().empty());
   CHECK_GT(blockHash.lastValidBlockHeight, 0);
 }
+TEST_CASE("MangoAccount is correctly created") {
+  const std::string& key = "9aWg1jhgRzGRmYWLbTrorCFE7BQbaz2dE5nYKmqeLGCW";
+  auto connection = solana::rpc::Connection(mango_v3::DEVNET.endpoint);
+  const auto& mangoAccountInfo =
+      connection.getAccountInfo<mango_v3::MangoAccountInfo>(key);
+  // Test prefetched account info
+  REQUIRE_NOTHROW(mango_v3::MangoAccount::from(mangoAccountInfo));
+  const auto& mangoAccountInfo_ =
+      connection.getAccountInfo<mango_v3::MangoAccountInfo>(key);
+  const auto& mangoAccount = mango_v3::MangoAccount::from(mangoAccountInfo_);
+  CHECK(!std::is_null_pointer<decltype(mangoAccount)>::value);
```

Comment:
i'm not sure what this check does

Reply:
The returned `unique_ptr` isn't a `nullptr_t`. Unlikely,  that it will be null without an exception happening( e.g some copy failed) which will just terminate the test but wanted to just be extra strict and validate everything.

Reply:
yeah but this is a type check, the type checker doesn't validate runtime errors

Reply:
you need to use CHECK(mangoAccount) to verify it's not null